src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -435,7 +435,7 @@ type WebAppConfig =
                   Site = this.ResourceId
                   Tags = this.Tags
                   AutoSwapSlotName = kvp.Value.AutoSwapSlotName
-                  AppSettings = cfg.AppSettings
+                  AppSettings = Map.fold (fun settings key value -> Map.add key value settings) this.Settings cfg.AppSettings // AppService settings first, then override with slot settings.
```

Comment:
I think there is a Map.merge method which does this (somewhere in the Farmer source)

Reply:
Ah - hadn't looked through the code in detail. Looks like it takes a list rather than a map, but the `Map.toList` is nicer to read - will update.